kubectl-minio/cmd/version.go

```diff
@@ -64,6 +75,35 @@ func newVersionCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 
 // run initializes local config and installs MinIO Operator to Kubernetes cluster.
 func (o *operatorVersionCmd) run() error {
-	fmt.Println(version)
+	fmt.Println("Kubectl-Plugin Version:", version)
+	cfg := config.GetConfigOrDie()
+	// If config is passed as a flag use that instead
+	k8sClient, err := client.New(cfg, client.Options{})
+	if err != nil {
+		return err
+	}
+	deployList := &v1.DeploymentList{}
+	listOpt := &client.ListOptions{}
+	client.MatchingLabels{"app.kubernetes.io/name": "minio-operator"}.ApplyToList(listOpt)
```

Comment:
The label value is `operator` and not `minio-operator` I feel.
After below change it works as expected
```
client.MatchingLabels{"app.kubernetes.io/name": "operator"}.ApplyToList(listOpt)
```

Reply:
This change still missing

Reply:
Not really. It will set by helm. It will name by 
```template
{{- define "minio-operator.name" -}}
  {{- default .Chart.Name | trunc 63 | trimSuffix "-" -}}
{{- end -}}
{{- define "minio-operator.selectorLabels" -}}
app.kubernetes.io/name: {{ include "minio-operator.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
{{- end -}}
```

Reply:
ok

Reply:
I did a fresh deployment of kind cluster and deployed minio operator using `./kubectl-minio init` but still the label value shows as `operator`
```
$ kubectl get deployments/minio-operator -n minio-operator -oyaml | yq '.metadata.labels."app.kubernetes.io/name"'
operator
```
In case of helm we can set the custom values but in case we deploy operator using plugin, the value default to `operator`. So is it fine to hard code value and check?